src/js/chatWindow.js

```diff
@@ -1,8 +1,20 @@
-export const addMessageToChatHistory = (m) => {
+const guardOptions = (options) => {
+  if (options.messageType !== 'outgoing' && options.messageType !== 'incoming') {
+    return false;
+  }
+
+  return true;
+};
+
+export const addMessageToChatHistory = (m, options = { messageType: 'outgoing' }) => {
```

Comment:
is there a more descriptive variable name that we can use for `m`?

Reply:
and I changed this one to "message"!